### PR TITLE
Add release notes for v2.9.4

### PR DESCRIPTION
**Pull request by nporter-adbe**

### CODE REVIEW

**Review by yangyansong-adbe — APPROVED** [review]

File: foundation-extensions/mobile-core/mobile-core-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+### June 22, 2021
```
Comment (yangyansong-adbe) [inline foundation-extensions/mobile-core/mobile-core-release-notes.md:3]: should update it to -> `## June 22, 2021`

File: foundation-extensions/mobile-core/mobile-core-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
+#### iOS Core 2.9.3
```
Comment (yangyansong-adbe) [inline foundation-extensions/mobile-core/mobile-core-release-notes.md:5]: `### iOS Core 2.9.3`

File: release-notes/README.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+### June 22, 2021
```
Comment (yangyansong-adbe) [inline release-notes/README.md]: ditto

File: release-notes/README.md [hunk 4]
```diff
@@ -4,6 +4,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+
+#### iOS Core 2.9.3
```
Comment (yangyansong-adbe) [inline release-notes/README.md:9]: ditto

File: foundation-extensions/mobile-core/mobile-core-release-notes.md (hunk 2 above)
Comment (nporter-adbe) [reply foundation-extensions/mobile-core/mobile-core-release-notes.md:5]: we may have done this for previous releases, that is use the wrong tag size

Comment (yangyansong-adbe) [reply foundation-extensions/mobile-core/mobile-core-release-notes.md]: I just compared it to the previous section not the older ones, yes, we should unify the heading format.

File: foundation-extensions/mobile-core/mobile-core-release-notes.md [hunk 5]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## June 22, 2021
+
+### iOS Core 2.9.3
+
+* Fixed a Rules Engine bug affecting strings that contain the `&` token
```
Comment (emdobrin) [inline foundation-extensions/mobile-core/mobile-core-release-notes.md:7]: that contain the `&` character (same for Readme)

Comment (emdobrin) [reply foundation-extensions/mobile-core/mobile-core-release-notes.md:7]: Let's also add periods at the end of each line.

**Pull request merged**